ui/src/app/views/account-creation/component.ts

```diff
@@ -65,8 +66,11 @@ export class AccountCreationComponent {
     this.profileService.createAccount(request).subscribe(() => {
       this.creatingAcccount = false;
       this.accountCreated = true;
-    }, () => {
+    }, (e) => {
       this.creatingAcccount = false;
+      if (e.status === 409) {
+        this.conflictError = true;
+      }
```

Comment:
This behavior is fine and definitely an improvement! Eventually we might want to call this check independently before the user clicks the create account button (which would help prevent an avoidable error by guiding the user to pick am available username before submitting)